mm2src/mm2_main/src/mm2_tests.rs

```diff
@@ -7528,3 +7528,150 @@ fn test_no_login() {
     .unwrap();
     assert!(version.0.is_success(), "!version: {}", version.1);
 }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_gui_storage() {
```

Comment:
How about splitting this huge integration test like one test function for each functionality, or at least grouping them like `test_gui_storage_account_functionality`, `test_gui_storage_coin_functionality` ? Otherwise this function will keep growing as the gui storage gets more functionality.

Reply:
Will do, thanks

Reply:
Done